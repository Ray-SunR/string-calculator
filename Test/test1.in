r a abcde
la
wa

r b abc
lb
wb

s c a b
lc
wc

e a
m a 2 b
la
wa

e a
p a b 2
la
wa

e a
n a cde
la
wa

lc
wc
e c

t c a fg
lc
wc

f d
ld
wd

e a
s a d b
la
wa
ea

e b
e c
e d
q
