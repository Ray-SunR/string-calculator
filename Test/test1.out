5
abcde
3
abc
8
abcdeabc
6
abcabc
6
abcabc
3
cde
8
abcdeabc
5
cdefg
0

3
abc
